elements/tagging-question/tagging-question.js

```diff
@@ -65,7 +65,7 @@ class TaggingQuestion extends QuestionElement(DDD) {
           border: var(--ddd-border-sm);
           border-radius: var(--ddd-radius-sm);
           box-sizing: border-box;
-          background-color: light-dark(
+          background-color: (
```

Comment:
if it's not changing between light and dark in this way then it can just be

`background-color: var(--ddd-theme-default-coalyGray);`